frontend/app/pages/index.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import { useLogin } from '~/composables/useLogin'
+import type { LoginResponse } from '~/composables/useLogin'
+
+const { login } = useLogin();
+
+const state = reactive({
+    email: '',
+    password: ''
+})
+
+async function submit() {
+    const response: LoginResponse = await login(state.email, state.password);
+
+    if (response.succes === true) {
```

Comment:
同様です。
```
    if (response.success === true) {
```

Reply:
もっというと true であるなら ifの中ではその値を処理できるので、次のように書いても問題なく動作すると思います。一度確認してみてください。
```suggestion
    if (response.success) {
```